lib/app.js

```diff
@@ -28,29 +28,100 @@ app.get('/api/test', (req, res) => {
   });
 });
 
-app.get('/characters', async(req, res) => {
-  try {
-    const data = await client.query('SELECT * from characters');
+// app.get('/characters', async(req, res) => {
+//   try {
+//     const data = await client.query('SELECT * from characters');
     
-    res.json(data.rows);
-  } catch(e) {
+//     res.json(data.rows);
+//   } catch(e) {
     
+//     res.status(500).json({ error: e.message });
+//   }
+// });
+
+app.post('/characters', async(req, res)=>{
+  
+  try {
+    const data = await client.query(`
+    INSERT INTO characters(
+      id,
```

Comment:
Remember that SQL handles the addition of IDs so you never need to `INSERT` an ID into your database unless it is a separate reference ID used for a different table.